### PR TITLE
Update app service params after provisioning

### PR DESCRIPTION
**Pull request by santaduda**
After app services are provisioned the app service parameters and KeyVault values are set.

### CODE REVIEW

File: create-env.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def show_help():
         subscription_id = "61a3b0f8-bf9b-4f99-89fa-f18c203c6086"
         resource_client = ResourceManagementClient(credential, subscription_id)
         web_client = WebSiteManagementClient(
-            credential=DefaultAzureCredential(),
+            credential=DefaultAzureCredential(exclude_shared_token_cache_credential=True),
```
Comment (rdevalap) [inline create-env.py:53]: I do not have this issue. Can you rollback this? Keep it DefaultAzureCredential?

**Review by rdevalap — CHANGES_REQUESTED** [review]
Hi @santaduda , As discussed,
1. Can you add the parameters for all app services?
2. Can you comment the line `KeyVault.substitue_expression(microservices_config)` in `diagnostic-settings.py`

File: create-env.py (hunk 1 above)
Comment (rdevalap) [reply create-env.py:53]: Also, test it from the github workflow?

Comment (rdevalap) [reply create-env.py:53]: Also, remove the "After" in Keyvault.substitute?

**Comment by santaduda** [comment]
1. Added all microservices and application parameters
2. Removed exclude_shared_token_cache_credential=True in DefaultAzureCredential()
3. Commented line : KeyVault.substitue_expression(microservices_config) in diagnostic-settings.py

**Review by rdevalap — APPROVED** [review]
Tested the code. Works well. 
Provisions app services, configures health check, diagnostic settings enabled

**Pull request merged**